packages/block-editor/src/components/writing-flow/utils.js

```diff
@@ -0,0 +1,17 @@
+/**
+ *
+ * Detects if element is a form element.
+ *
+ * @param {Element} element The element to check.
+ *
+ * @return {boolean} True if form element and false otherwise.
+ */
+export function isFormElement( element ) {
```

Comment:
I think this should probably end up in the `@wordpress/dom` package. That already has some quite similar functions (`isHTMLInputElement`, `isInputOrTextArea`, `isNumberInput`, `isTextField`).

I don't know why it has so many similar functions, but there doesn't seem to be an `isFormElement` yet.

Once moved the `isInputOrTextArea` function could be used in the implementation of this function.

Reply:
@talldan Moved the function to wordpress/dom.

Reply:
Thanks!